twitter_account.rb

```diff
@@ -1,10 +1,14 @@
-require 'active_record'
 require 'mysql2'
 
-ActiveRecord::Base.establish_connection(
-    "adapter" =>"mysql2",
-    "database" => "twitter_analytics_acquirer"
-)
+class TwitterAccount
+  @client = Mysql2::Client.new(host: "localhost", username: "root", password: '', database: 'twitter_analytics_acquirer')
 
-class TwitterAccount < ActiveRecord::Base
+  def self.all
+    @client.query("SELECT `twitter_accounts`.* FROM `twitter_accounts`").each
+  end
+
+  def self.update(column, value, id)
+    escaped_value =  @client.escape(("#{value}"))
+    @client.query("UPDATE `twitter_accounts` SET #{column} = '#{escaped_value}' WHERE  id = #{id}")
+  end
```

Comment:
It may be better to define  `update_cookies` .
Because other columns can fail to update value, because value to escape is up to his column.

EX) column == id, id is int and not to be escaped.

Reply:
And it may be better to be instance method.
But 'creating object' is needed in that case.(It is a little bothersome only to deal with method.)